src/main/java/samza/examples/sql/samza-sql-stream-table-join/src/main/sql/samza.sql

```diff
@@ -0,0 +1,6 @@
+insert into log.outputStream
```

Comment:
Could we replace this with:

-- Enrich PageViewEvent with member profile data

INSERT INTO kafka.tracking.EnrichedPageViewEvent
   SELECT *
   FROM kafka.PageViewEvent as pv
      JOIN kafka.ProfileChanges.`$table` as p
          ON pv.memberId = p.memberId

Reply:
$table should have back ticks around it.

Reply:
done